core/storage/feedback/gae_models.py

```diff
@@ -64,11 +63,12 @@ class FeedbackThreadModel(base_models.BaseModel):
     subject = ndb.StringProperty(indexed=False)
     # Summary text of the thread.
     summary = ndb.TextProperty(indexed=False)
+    # Specifies whether this thread has a related learner suggestion.
+    has_suggestion = ndb.BooleanProperty(indexed=False)  
```

Comment:
This will affect existing threads, so I suggest adding a "default=False" as well.